todo-list/fn_and_obj.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+'''
+    File name: fn_and_obj.py
+    Author: Henry Letton
+    Date created: 2020-08-14
+    Python Version: 3.7
+    Desciption: This program creates functions and objects required for main.py
+'''
+
+# Import any required modules
+from datetime import datetime
+import pickle
+
+
+# Using the pickle module, a to do list can be saved to disc (in wd)
+def save_list_to_disc(todo_list):
+
+    outfile = open('ToDoListSave','wb')
+    pickle.dump(todo_list,outfile)
+    outfile.close()
+
+
+# If a previous list exists (in wd), the user can decide whether to use it or start afresh
+def load_prev_list():
+    
+    try:
+        infile = open('ToDoListSave','rb')
+        todo_list = pickle.load(infile)
```

Comment:
Nice use of pickling!
Just one thing to bear in mind is that it risks remote code execution (i.e. I coud put something malicious in that will get loaded, like sending me your credit card numbers). 
Completely fine for this task, but something to bear in mind in future if there's any risk of someone having tampered with the pickle file you're loading.